IMU_v2/IMU_v2.ino

```diff
@@ -0,0 +1,134 @@
+//Component: Yost 3-Space™ LX Embedded part number TSS-LX
```

Comment:
- I think you can delete this whole folder if the `test-i2c` folder is the one you want to use